tests/test-land-patterns.stanza

```diff
@@ -0,0 +1,51 @@
+#use-added-syntax(jitx)
+defpackage ocdb/test/land-patterns:
+  import core
+  import collections
+  import jitx
+  import jitx/commands
+  import ocdb/defaults
+  import ocdb/design-vars
+  import ocdb/land-patterns
+  import ocdb/tolerance
+  import ocdb/box-symbol
+  import ocdb/rules
+
+pcb-component example-1:
+  port p: pin[[1, 2]]
+  val lp = ipc-compliant-two-pin-landpattern(
+    Toleranced(1.0, 0.05, 0.06)  ; length of the component
```

Comment:
I have an `Interval` structure defined: see `public defstruct Interval`

Reply:
`Interval` doesn't cover the same use cases, the `typ` or `nominal` value is not strictly the center of the range. It may also be described in percentages.